benches/distribution/cauchy.rs

```diff
@@ -0,0 +1,64 @@
+use probability::prelude::*;
+use test::{black_box, Bencher};
+
+#[bench]
+fn distribution(bencher: &mut Bencher) {
+    let d = Cauchy::new(0.0, 1.0);
+    let x = Independent(&d, &mut source::default())
+        .take(1000)
+        .collect::<Vec<_>>();
+
+    bencher.iter(|| black_box(x.iter().map(|&x| d.distribution(x)).collect::<Vec<_>>()));
+}
+
+#[bench]
+fn inverse(bencher: &mut Bencher) {
+    let d = Cauchy::new(0.0, 1.0);
+    let p = Independent(&Uniform::new(0.0, 1.0), &mut source::default())
+        .take(1000)
+        .collect::<Vec<_>>();
+
+    bencher.iter(|| black_box(p.iter().map(|&p| d.inverse(p)).collect::<Vec<_>>()));
+}
+
+/// Benchmark sampling via the implemented method.
+#[bench]
+fn sample(bencher: &mut Bencher) {
+    let mut source = source::Xorshift128Plus::new([42, 69]);
+    let d = Cauchy::new(0.0, 1.0);
+
+    bencher.iter(|| black_box(d.sample(&mut source)));
+}
+
+/// Benchmark sampling via the direct method (applying the quantile function to
```

Comment:
I am not sure why this and the next one are needed. There is an implementation, and it is benchmarked. Alternatives with no API presence are a bit besides the point. What are your thoughts?

Reply:
I used these benchmarks to decide with which implementation to go, and then I left them in to document for other contributors why I ended up choosing the less obvious implementation. But I'm happy to remove the benchmarks if you'd prefer that.